core/src/main/java/com/datastax/oss/driver/internal/core/control/ControlConnection.java

```diff
@@ -226,7 +226,20 @@ private void processStatusChange(Event event) {
 
   private void processSchemaChange(Event event) {
     SchemaChangeEvent sce = (SchemaChangeEvent) event;
-    context.getMetadataManager().refreshSchema(sce.keyspace, false, false);
+    context
+        .getMetadataManager()
+        .refreshSchema(sce.keyspace, false, false)
+        .whenComplete(
```

Comment:
Wouldn't be simpler to use `exceptionally`?

Reply:
It's equivalent. But `exceptionally` creates another future with a fallback value, which is not needed here.
I did use it in `CqlRequestHandler` though, because I needed a future.